status/status.go

```diff
@@ -131,6 +131,16 @@ func FromError(err error) (s *Status, ok bool) {
 	return nil, false
 }
 
+// Convert returns a Status representing err.  If err was not produced by this
+// package, it returns a Status with an Unknown code and a message of
+// err.Error().
+func Convert(err error) *Status {
+	if s, ok := FromError(err); ok {
+		return s
+	}
+	return &Status{s: &spb.Status{Code: 0, Message: err.Error()}}
```

Comment:
Code being set to an integer directly should be avoided.  `0` is `OK`, which is not what we want here.

However, can you make the other change recommended in #1378?  I.e. change `FromError`'s final return to:

```go
return New(codes.Unknown, err.Error()), false
```

and then unconditionally return the first value from `FromError` here.

Reply:
That's much nicer. Will do.